tools/export.py

```diff
@@ -40,6 +41,20 @@ def get_min_opset_version():
     return 10 if version.parse(torch.__version__) < version.parse('1.7.0') else 11
 
 
+def patch_model_for_alt_ssd_export(model):
+    model._export_mode = False
+    model.onnx_export = onnx_export.__get__(model)
+    model.save_img_metas = save_img_metas.__get__(model)
+    model.forward = forward.__get__(model)
+    model.forward_export = forward_export_detector.__get__(model)
+    model.bbox_head.export_forward = export_forward_ssd_head.__get__(model.bbox_head)
+    model.bbox_head._prepare_cls_scores_bbox_preds = prepare_cls_scores_bbox_preds_ssd_head.__get__(model.bbox_head)
+
+
+def patch_nncf_model_for_alt_ssd_export(model):
+    model.onnx_export = onnx_export.__get__(model)
```

Comment:
What's the purpose of this? Does NNCF model has `save_img_metas` method?

Reply:
Without it onnx_export works with not wrapped model, as result exported onnx model does not contain nncf modules.
As i understand save_img_metas is not required for this case.